formats/dot/internal/errors/errors.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// generated by gocc; DO NOT EDIT.
+// Code generated by gocc; DO NOT EDIT.
```

Comment:
Large diffs get hidden as well. It's unfortunate gocc doesn't give greater flexibility (I've used ragel which does this kind of thing very simply). I will think about this.

Good spotting on the `\n`. I'll fix that. Thanks.